### PR TITLE
feat(integration): support retrieve xml format response

### PR DESCRIPTION
**Pull request by smallsheeeep**
#### What type of PR is this?
type-feature
module-integration

#### What this PR does / why we need it:
Support retrieve xml format response

#### Which issue(s) this PR fixes:
Closing #313 

#### Special notes for your reviewer:
Unit test has covered

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/integration/model/IntegrationProperties.java [hunk 1]
```diff
@@ -53,13 +53,14 @@ public static class ApiProperties {
         private Body body;
         private boolean requestEncrypted = false;
         private boolean responseEncrypted = false;
+        private ResponseType responseType = ResponseType.JSON;
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/integration/model/IntegrationProperties.java:56]: it is not 'response type', but format of the response body.

similar concept may 'contentType'.

however, we may got content type from response header 'Content-Type', so if the content type is not defined,
ODC may detect from response header.

Comment (smallsheeeep) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/integration/model/IntegrationProperties.java:56]: I don't think we need to pay extra attention to the `Content-type` in response header. (Yes we can, but not necessary)

Apache http-core will parse response to `String` based on the Content-type automatically. So we should not care about different type of response content, just process the `String` according to the format specified by the integrator.

Comment (smallsheeeep) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/integration/model/IntegrationProperties.java]: Rename `responseType` to `responseContentType`

**Review by yizhouxw — APPROVED** [review]
LGTM

**Pull request merged**